main/Logger/Exports.cpp

```diff
@@ -0,0 +1,14 @@
+#include "FileLogger.h"
```

Comment:
We have ATLTRACE and RELTRACE macros for logging to DEBUG output (which can be picked up by DebugView)